terraform/table_terraform_resource.go

```diff
@@ -36,11 +36,21 @@ func tableTerraformResource(ctx context.Context) *plugin.Table {
 				Description: "Resource type.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
+				Name:        "mode",
```

Comment:
Are these new columns something only in state files, or did we miss them when adding support for configuration files?